Packs/MicrosoftExchangeOnline/Integrations/EwsExtensionEXOPowershellV3/EwsExtensionEXOPowershellV3.ps1

```diff
@@ -1760,6 +1764,22 @@ function SetJunkRulesCommand([ExchangeOnlinePowershellV3Client]$client, [hashtab
     return $human_readable, $entry_context, $raw_response
 }
 
+function  CreateAddAndRemoveSections([string[]]$items_to_add, [string[]]$items_to_remove ){
+    if ((-not [string]::IsNullOrEmpty($items_to_add)) -and (-not [string]::IsNullOrEmpty($items_to_remove))){
```

Comment:
kind of ugly with treating this as 3 cases. Can we say "add it if it exists to an object"? like we would do with python

something like 
```

$object = @{}

if (-not [string]::IsNullOrEmpty($items_to_add)) {
    $object.Add = $items_to_add
}

if (-not [string]::IsNullOrEmpty($items_to_remove)) {
    $object.Remove = $items_to_remove
}`
```